bridge/client/app/_views/ktb-settings-view/ktb-project-settings/ktb-project-settings.component.ts

```diff
@@ -68,22 +65,82 @@ export class KtbProjectSettingsComponent implements OnInit, OnDestroy, PendingCh
 
   public message = 'You have pending changes. Make sure to save your data before you continue.';
   public unsavedDialogState: DialogState = null;
-  public resourceServiceEnabled?: boolean;
+
+  public resourceServiceEnabled$ = this.featureFlagService.featureFlags$.pipe(
+    map((featureFlags: IClientFeatureFlags) => featureFlags.RESOURCE_SERVICE_ENABLED)
+  );
+
+  public projectName$: Observable<string | null> = this.route.paramMap.pipe(
+    map((params) => params.get('projectName')),
+    tap((projectName) => {
+      if (projectName) {
+        this.projectName = projectName;
+        this.projectDeletionData = {
```

Comment:
Is it possible to get rid of the internal states `this.projectName` and `this.projectDeletionData`?

Reply:
Can I trigger `this.notificationsService.addNotification` somehow within the template? Or should I `.subscribe` in this case to get the values?

Reply:
I moved the `projectDeletionData` to `state$`, but for `projectName` not sure if I should rather go with `state$.subscribe`, leave the `this.projectName` or maybe you have another suggestion?

Reply:
My bad, actually there was a very easy way also to get rid of `this.projectName`. Fixed it 👍🏻